test_mask_upload.py

```
@@ -0,0 +1,148 @@
+            "cfg_scale": 7.0,
+            "sampler_name": "euler",
+            "scheduler": "normal",
+            "width": 512,
+            "height": 512,
+            "batch_size": 1,
+            "seed": 42
+        }
+
+        # Create multipart form data
+        files = {
+            'image': ('test_image.png', open(image_path, 'rb'), 'image/png'),
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)